src/deploy/table_report.sql

```diff
@@ -0,0 +1,24 @@
+-- Deploy maevsi:table_report to pg
+
+BEGIN;
+
+CREATE TABLE maevsi.report (
+    id              UUID PRIMARY KEY NOT NULL DEFAULT gen_random_uuid(),
+    reporter_id     UUID NOT NULL REFERENCES maevsi.account(id),
+    reason          TEXT NOT NULL,
```

Comment:
User input should be limited like this:
```suggestion
    reason          TEXT NOT NULL CHECK (char_length("reason") > 0 AND char_length("reason") < 2000),,
```